src/elm/Page/Community.elm

```diff
@@ -1006,7 +1006,7 @@ viewTransfer loggedIn model transfer =
     in
     div [ class "border-b last:border-b-0 border-gray-500 flex flex-wrap items-start p-4" ]
```

Comment:
Please add a link here. Change the `div` to a `a` tag and add the `href` function to the array of attributes:

```elm
    a
        [ class "border-b last:border-b-0 border-gray-500 flex flex-wrap items-start p-4"
        , Route.href (Route.Transfer transfer.id)
        ]
        [ div [ class "flex justify-between w-full" ]

. . . 
```